app/views/users/show.html.erb

```diff
@@ -36,6 +36,18 @@
             <%= c.user_count%>
           </div>
         </div>
+        <div class='col'>
+          <div class='course-details_title'>
+            Role
+          </div>
+          <div class='course-details_value'>
+            <% r = c.courses_users %>
+            <% r.each do |p| %>
+              <% @pass = p.role %>
+            <%end%>
+            <%= role(@pass) %>
```

Comment:
This also might be a good time to refactor this entire loop for the course into a partial and convert it to haml. (Optional for this pull request.)